### PR TITLE
docs: add unicode abbreviations in no-irregular-whitespace rule

### PR DESCRIPTION
**Pull request by alixroyere**
#### Prerequisites checklist

- [X] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
I added unicode abbreviations in no-irregular-whitespace rule documentation. This is usefull because these abbreviations often appear in IDEs instead of unicode values/code for these invisible characters.

#### Is there anything you'd like reviewers to focus on?
Having a nice day

### CODE REVIEW

File: docs/src/rules/no-irregular-whitespace.md [hunk 1]
```diff
@@ -39,27 +39,27 @@ This rule disallows the following characters except where the options allow:
...
+\u200A - Hair Space - <HSP>
 \u200B - Zero Width Space - <ZWSP>
-\u2028 - Line Separator
-\u2029 - Paragraph Separator
-\u202F - Narrow No-Break Space
-\u205f - Medium Mathematical Space
-\u3000 - Ideographic Space
+\u2028 - Line Separator - <LSEP>
+\u2029 - Paragraph Separator - <
PSEP>
```
Comment (mdjermanovic) [inline docs/src/rules/no-irregular-whitespace.md:59]:
```suggestion
\u2028 - Line Separator - <LS> - <LSEP>
\u2029 - Paragraph Separator - <PS> - <PSEP>
```

**Review by mdjermanovic — COMMENTED** [review]
Thanks for the PR! I have only one small suggestion.

File: docs/src/rules/no-irregular-whitespace.md [hunk 2]
```diff
@@ -39,27 +39,27 @@ This rule disallows the following characters except where the options allow:
 \u000B - Line Tabulation (\v) - <VT>
 \u000C - Form Feed (\f) - <FF>
 \u00A0 - No-Break Space - <NBSP>
-\u0085 - Next Line
-\u1680 - Ogham Space Mark
+\u0085 - Next Line - <NEL>
+\u1680 - Ogham Space Mark - <OGSP>
 \u180E - Mongolian Vowel Separator - <MVS>
```
Comment (amareshsm) [inline docs/src/rules/no-irregular-whitespace.md:44]:
```suggestion
\u180E - Mongolian Vowel Separator - <MVS> - <MVSEP>
```

Comment (mdjermanovic) [reply docs/src/rules/no-irregular-whitespace.md:44]: @amareshsm is MVSEP a common abbreviation for this character? I couldn't find it in any documents.

Comment (amareshsm) [reply docs/src/rules/no-irregular-whitespace.md:44]: In documentation dealing with Mongolian script, a more explicit term like <MVSEP> could provide better clarity over the standard <MVS> but is not considered a common abbreviation. let's ignore this comment.

**Review by amareshsm — APPROVED** [review]
LGTM. Waiting for Milos's review

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**